docs/proposals/20200511-clusterctl-extensible-template-processing.md

```diff
@@ -238,7 +238,7 @@ libraries so the issue of support should be solved with this contract.
   a single yaml file. These artifacts will need to be “grouped” together to
   support current retrieval mechanisms. Currently, `clusterctl config cluster`
   retrieves templates from multiple sources such as ConfigMaps within a
-  cluster, URL, Github Repository, Local Repository and even the overrides
+  cluster, URL, Github Repository, Local Repository and even the overrides'
```

Comment:
Not sure this apostrophe is required here, what's the motivation on this one? I think `overrides directory` forms one noun here rather than `overrides` owning the directory 🤔

Reply:
Thanks for your reviews @JoelSpeed , you are right, I will revise it.